### PR TITLE
fix: improve dashboard/show page's performance and make /home redirect to default collection page

### PR DESCRIPTION
**Pull request by Code-Beast**
## Context
Two issues:
1. Default collection is created after other previously created collections, so if we want to redirect to the default collection dashboard page, we can no longer use last collection in the returned collection array.
2. Currently, in dashboard/show page, in order to enable events-thumbnail component to generate events graph, route.js has to fetch events as well as their builds (for generating duration information) which is very inefficient and can be improved by using pipeline metrics which contains event duration info that is computed in the backend.

## Objective
1. Improve dashboard/show page's performance
2. Redirect /home to default collection page
3. Fix branch info overflow issue in pipeline card

## References
[UI feat-480](https://github.com/screwdriver-cd/ui/pull/480)
[Models fix-397](https://github.com/screwdriver-cd/models/pull/397)

## License
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/events-thumbnail/component.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export default Component.extend({
       .attr('y', event => 40 - y(event.duration));
 
     bars.each(function addColor(bar) {
-      this.classList.add(bar.statusColor.toLowerCase());
+      this.classList.add(bar.statusColor);
```
Comment (adong) [inline app/components/events-thumbnail/component.js]: Should we keep the logics of converting to lowercases regardless? @Code-Beast ?

File: app/components/pipeline-card/styles.scss [hunk 2]
```diff
@@ -93,34 +93,41 @@
   }
 
   .branch-info {
-    display: flex;
-    justify-content: space-between;
+    // display: flex;
```
Comment (tkyi) [inline app/components/pipeline-card/styles.scss]: commented code

File: app/components/pipeline-card/styles.scss [hunk 3]
```diff
@@ -93,34 +93,41 @@
...
-      & a {
-        width: 78%;
-      }
-
-      & > div {
-        width: 22%;
-      }
-    }
-  }
+  // @media screen and (max-width: 1440px) {
```
Comment (tkyi) [inline app/components/pipeline-card/styles.scss]: same here

File: app/components/events-thumbnail/component.js (hunk 1 above)
Comment (Code-Beast) [reply app/components/events-thumbnail/component.js]: statusColor is always lowercase in my logic

**Pull request merged**